### PR TITLE
fix: replaced process with typeof window since it's deprecated

### PR DESCRIPTION
**Pull request by thegeeko**
# replaced process.browser with typeof window

- [x] I reviewed linter warnings + errors, resolved formatting, types and other issues related to my work
- [x] The PR title follows the convention we established [conventional-commit](https://www.conventionalcommits.org/en/v1.0.0/)
- [x] I performed a functional test on my final commit

---
replaced process.browser with typeof window since it's depreciated refer to this [comment](https://github.com/vercel/next.js/issues/5354#issuecomment-520305040)

---
💯

### CODE REVIEW

**Comment by nexxeln** [comment]
The tRPC docs use this, so we're using it too. Are you sure this works with tRPC?

**Comment by thegeeko** [comment]
I'm not sure but I can't see why tRPC could have a problem with it .. I'll do some testing

**Comment by thegeeko** [comment]
it works just with and without tRPC ssr enabled

**Comment by nexxeln** [comment]
We already have this, no need to check for the same condition twice. Thank you but we'll be closing this.

File: template/page-studs/_app/with-trpc.tsx [hunk 1]
```diff
@@ -13,7 +13,7 @@ const getBaseUrl = () => {
   if (typeof window !== "undefined") {
     return "";
   }
```
Comment (asheeeshh) [inline template/page-studs/_app/with-trpc.tsx:15]: here ^

Comment (asheeeshh) [inline template/page-studs/_app/with-trpc.tsx:15]: here ^

Comment (thegeeko) [reply template/page-studs/_app/with-trpc.tsx:15]: idk how I missed this .. sry .. but then isn't the next process.browser technically the same thing shouldn't u  remove it?? or  it's there just to be sure

**Pull request closed without merging**